scripts/pylib/twister/twisterlib/testinstance.py

```diff
@@ -201,7 +201,6 @@ def setup_handler(self, env):
         if handler:
             handler.options = options
             handler.generator_cmd = env.generator_cmd
-            handler.generator = env.generator
```

Comment:
should this value be used instead of removing, e.g. for logging ?

Reply:
I don't think so, as this is not used. Note that now, `runner.py` uses `environment.py`'s `generator` field 
https://github.com/zephyrproject-rtos/zephyr/blob/78150c8e1b2c07537ba35a05d9428983f1968f99/scripts/pylib/twister/twisterlib/runner.py#L337-L343 which is logged already 
https://github.com/zephyrproject-rtos/zephyr/blob/78150c8e1b2c07537ba35a05d9428983f1968f99/scripts/pylib/twister/twisterlib/environment.py#L884-L890